src/SwedbankPay.Episerver.Checkout/SwedbankPayCheckoutGateway.cs

```diff
@@ -42,15 +59,81 @@ public PaymentProcessingResult ProcessPayment(IOrderGroup orderGroup, IPayment p
                 : PaymentProcessingResult.CreateUnsuccessfulResult(message);
         }
 
-        public override bool ProcessPayment(Payment payment, ref string message)
+        /// <summary>
+        /// Processes a payment. Can be used for both positive and negative transactions.
+        /// </summary>
+        /// <param name="payment">The payment to be processed.</param>
+        /// <param name="message">The message passed back, most often an error if transaction failed.</param>
+        /// <returns><c>True</c> if process successfully; otherwise <c>False</c>.</returns>
+        public bool ProcessPayment(Payment payment, ref string message)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider refactoring this method in order to remove the need for this 'ref' modifier.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5328257)